src/xcode/ENA/ENA/Source/Scenes/Events/Cells/EventTableViewCell.swift

```diff
@@ -89,6 +89,8 @@ class EventTableViewCell: UITableViewCell {
 
 		titleLabel.text = cellModel.title
 		addressLabel.text = cellModel.address
+		addressLabel.accessibilityLabel = String(format: AppStrings.TraceLocations.Configuration.address, cellModel.address)
+		addressLabel.accessibilityIdentifier = String(format: AppStrings.TraceLocations.Configuration.address, cellModel.address)
```

Comment:
```suggestion
```

The identifier is just used for the UI test, I don't think it makes sense to put the same value as the accessibility label in there.

Reply:
There are two different identifiers, I have created a new one, so not using the one for the UI Tests.